src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -103,8 +113,8 @@ public Boolean getAddTestResults() {
         return addTestResults == null ? Boolean.valueOf(false) : addTestResults;
     }
 
-    public Boolean getAddCoverageResults() {
-        return addCoverageResults == null ? Boolean.valueOf(false) : addCoverageResults;
+    public String getCoverageResultFilePath() {
+        return coverageResultFilePath == null ? "" : coverageResultFilePath;
```

Comment:
No point in doing this. You have to check for empty later might as well check for null `Strings.isNullOrEmpty()`.

Reply:
I'm only doing this because it is the standard that they set in other similar methods.

Reply:
I think it's better to keep it consistent